test/rewrite_clj/parser_test.cljc

```diff
@@ -1,10 +1,9 @@
 (ns ^{:doc "Tests for EDN parser."
       :author "Yannick Scherer"}
  rewrite-clj.parser-test
-  (:refer-clojure :exclude [read-string])
   (:require [clojure.string :as string]
-            [clojure.test :refer [deftest is]]
-            [clojure.tools.reader.edn :refer [read-string]]
+            [clojure.test :refer [deftest is testing]]
+            [clojure.tools.reader :as rdr]
```

Comment:
Do you need to use clojure.tools.reader instead of clojure.tools.reader.edn for your tests to be valid?

Reply:
My initial motivation to tweak this was to automatically test this statement in the docs: "Within reason, Clojure’s read-string and rewrite-clj’s sexpr functions should return equivalent Clojure forms".

So that meant using `clojure.core/read-string` instead of `clojure.tools.reader.edn/read-string`. This is because:
1. I interpreted "Clojure’s read-string" to mean `clojure.core/read-string`
2. Things like `'`@~` are not valid edn, so the tests would not make sense.

Since CLJS doesn't have `read-string`, I replaced `c.t.n` with `c.t.n.edn` for reason number 2.

The main downside is `*read-eval*` can be dangerous, so I tried to be as conservative as possible in setting that.

I think perhaps the test in `t-parsing-seqs` could also test against `clojure.core/read-string` in the `:default` case for reason number 1.

Reply:
Ah, ok, thanks for explaining, @frenchy64.

The thing is sci tests are currently failing.
Sci tests are a bit involved, would you like me to have a look-see?